packages/base/any/kernels/3.16-lts/patches/driver-i2c-bus-intel-ismt-netberg-aurora.patch

```diff
@@ -0,0 +1,16 @@
+diff -Nu a/drivers/i2c/busses/i2c-ismt.c b/drivers/i2c/busses/i2c-ismt.c
```

Comment:
@raymondhuey Lets remove this change from your pull request for the moment since it would affect other platforms and get the rest of your code merged. Then we can work separately on the issue you are describing.